### PR TITLE
Add Node.js implementation

### PR DESCRIPTION
**Pull request by selfisekai**

### CODE REVIEW

File: node/src/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  const cipher = crypto.createCipheriv(ALGORITHM, pass, Buffer.alloc(0));
+  let encrypted = cipher.update(cont, UTF8, BASE64);
+  encrypted += cipher.final(BASE64);
+  return encrypted;
+};
+
+/*
+const decode = (password, content) => {
+  const pass = toArrayBuffer(password);
+  const cont = toArrayBuffer(content);
```
Comment (mklkj) [inline node/src/index.js]: Simply don't convert content to array buffer:

```suggestion
  const cont = content;
```

File: node/src/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+declare module '@wulkanowy/qr-node' {
+  export function encode(password: string, content: string): string;
+  export function decode(password: string, content: string): string;
+}
```
Comment (mklkj) [inline node/src/index.d.ts]: Add \n at the end of file

File: node/package.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@wulkanowy/qr-node",
```
Comment (mklkj) [inline node/package.json]:
```suggestion
  "name": "@wulkanowy/qr",
```

File: node/yarn.lock [hunk 4]
```diff
@@ -0,0 +1,1642 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
[the remark this answers is not in the document]
Comment (selfisekai) [reply node/yarn.lock]: this file should be commited as per yarn docs: https://classic.yarnpkg.com/en/docs/yarn-lock/#toc-check-into-source-control

File: node/package.json (hunk 3 above)
Comment (selfisekai) [reply node/package.json]: intended - there could also be a browser implementation on ms npm, and they must be separate packages

**Pull request merged**